### ISSUE TITLE
Dice work, but no popup to tell me what I rolled

### ISSUE THREAD

**Issue by NeonAGC** [Theronguard/Emergency-Dice#17]
The dice function as intended, and the effects occur properly, except we have no popup telling us what we rolled, i.e 1-6. It was working in earlier versions but latest is bugged. Don't currently have a log to help in finding the issue, but I will include it when I manage to capture it

**Comment by Theronguard** [comment]
Which version are you using (both the mod and the game)?

**Comment by NeonAGC** [comment]
v50 for the game, and v1.120 for the mod. Although waking up I now see you've updated it to v1.121 so I'll give that one a shot and see if it works real quick

**Comment by NeonAGC** [comment]
Alright just tried it, and managed to get the log at long last


[Info   :Theronguard.EmergencyDice] [Debug] Player: NeonAGC rolled Swap
[Error  : Unity Log] MissingMethodException: Method not found: int .RoundManager.PlayRandomClip(UnityEngine.AudioSource,UnityEngine.AudioClip[],bool,single,int)
Stack trace:
LC_API.GameInterfaceAPI.Features.Player.ShowTipInternal (System.String header, System.String message, System.Single duration, System.Boolean isWarning, System.Boolean useSave, System.String prefsKey) (at <9c282cdfc7dc4cca9fcc5df099e02800>:IL_00AF)
LC_API.GameInterfaceAPI.Features.Player.ShowTip (System.String header, System.String message, System.Single duration, System.Boolean isWarning, System.Boolean useSave, System.String prefsKey) (at <9c282cdfc7dc4cca9fcc5df099e02800>:IL_0043)
LC_API.GameInterfaceAPI.Events.Patches.Internal.DisplayTipPatch.Prefix (System.String headerText, System.String bodyText, System.Boolean isWarning, System.Boolean useSave, System.String prefsKey) (at <9c282cdfc7dc4cca9fcc5df099e02800>:IL_0011)
(wrapper dynamic-method) HUDManager.DMD<HUDManager::DisplayTip>(HUDManager,string,string,bool,bool,string)
MysteryDice.Dice.EmergencyDie.Roll () (at <84fa60fb9357407ea532c855518e5491>:IL_00A8)
MysteryDice.Dice.DieBehaviour+<UseTimer>d__14.MoveNext () (at <84fa60fb9357407ea532c855518e5491>:IL_0130)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <e27997765c1848b09d8073e5d642717a>:IL_0026)

**Comment by NeonAGC** [comment]
^^ and yes it failed again, the logs there were in red (aka the error text)

**Comment by NeonAGC** [comment]
of course now that I think about it, that error log could just be because it looks like I rolled swap but I was in solo

but I have a short clip of the bug I just captured on outplayed, you can see there's no popup to tell me what I rolled (and yes, this is v1.121)

 https://outplayed.tv/media/j4dkma

**Comment by Theronguard** [comment]
I tested this just now. This is a conflict with LC_API mod. Disabling it fixes the issue. It seems it's a problem on LC_API's side so I'd either have to find some roundabout way to display messages or wait until LC_API is fixed. It could also be an issue on my part. I'll have to investigate it further tomorrow as I only glanced at the code before bed. Thanks for reporting the bug.

**Comment by NeonAGC** [comment]
Alright, I see. Thank you!

**Comment by Kimeunbe** [comment]
Is this problem solved?
We are experiencing the same problem.🥲

**Comment by Theronguard** [comment]
No. I took a look at LC_API code and it seems like it is not a problem with my mod. I'm working on a next version and I might find a workaround before the update, but I don't guarantee this. I'll try to open an issue on the LC-API's github.

**Comment by Theronguard** [comment]
I updated the mod to the 1.1.22 version. I added a fallback method that will print the effect info to the chat when the display tip method fails. Should work with LC_API now.

**Comment by Kimeunbe** [comment]
Alright, I see. Thank you! 🙂

**Comment by TheSilverWulf** [comment]
Same issue now, caused by GraphicAPI + Diversity.
